0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -36,67 +36,34 @@ The server used on the recipient's side to handle SPSP requests.
 
 #### Receiver
 
-The specific HTTP endpoint on the SPSP used for setting up a payment.
+The specific HTTPS endpoint on the SPSP used for setting up a payment.
 
 ## Overview
 
 ### Relation to Other Protocols
 
-SPSP is used for exchanging payment information before an ILP payment is initiated. It uses the receiver generated [Interledger Payment Requests](../0011-interledger-payment-request) over HTTP.
+SPSP is used for exchanging payment information before an ILP payment is initiated. The sender and recipient use the [Pre-Shared Key transport protocol(../0016-pre-shared-key/0016-pre-shared-key.md) to generate the condition and fulfillment. The recipient generates the shared secret to be used in PSK and communicates it to the sender over HTTPS.
```

Comment:
Is SPSP totally tied to PSK, or could you substitute another transport layer instead, e.g. in case you want a non-repudiable fulfillment?